Sources/web3swift/Utils/ENS/ENSResolver.swift

```diff
@@ -19,30 +19,13 @@ public extension ENS {
             case URI = 8
         }
 
-        public enum InterfaceName {
-            case addr
-            case name
-            case content
-            case ABI
-            case pubkey
-            case text
-
-            func hash() -> String {
-                switch self {
-                case .addr:
-                    return "0x3b3b57de"
-                case .name:
-                    return "0x691f3431"
-                case .content:
-                    return "0xbc1c58d1"
-                case .ABI:
-                    return "0x2203ab56"
-                case .pubkey:
-                    return "0xc8690233"
-                case .text:
-                    return "0x59d1d43c"
-                }
-            }
+        public enum InterfaceName: String {
+            case addr = "0x3b3b57de"
+            case name = "0x691f3431"
+            case content = "0xbc1c58d1"
+            case ABI = "0x2203ab56"
+            case pubkey = "0xc8690233"
+            case text = "0x59d1d43c"
```

Comment:
Personally, I prefer the previous implementation of the enum - as those are more flexible than assigning a type directly. I let others chime in on what they think though.

Reply:
I'm not in context of purpose of this enum, but I'd doubt about this change also, like it's a good practice to made swift enums without any raw type if it's not them main purpose. But I'd like to know whether the purpose of this one to provide such hashes or not, and how much those hashes are constant?

Reply:
Anyway I'm up for made hash computed variable rather function. This one definitely worth it.

Reply:
addressed in next commit